build.project.xml

```diff
@@ -62,8 +62,29 @@
         <setmysqlautoincrement />
         <phingcall target="writable-settings-folder" />
         <phingcall target="writable-settings-php" />
+
+        <!-- Avoid duplicates in settings.php, if the file exists. -->
+        <reflexive>
```

Comment:
Let's not hack this into the installer. This should be a separate Phing target, like all the other instances where we make a change to the settings file (e.g. `enable-config-read-only`, `disable-persistent-sessions-cookies` etc).

Also the location of the private files folder is environment specific so this should be in `settings.local.php` instead.